db/migrate/20220407063651_create_borrow_requets.rb

```diff
@@ -0,0 +1,13 @@
+class CreateBorrowRequets < ActiveRecord::Migration[6.0]
+  def change
+    create_table :borrow_requets do |t|
+      t.date :borrow_date
+      t.date :return_date
+      t.boolean :status
```

Comment:
integer https://betterprogramming.pub/how-to-use-enums-in-rails-6-87600e292476#:~:text=In%20Ruby%20on%20Rails%2C%20an,for%20enums%20in%20Rails%204.1.